application/briefkasten/dropbox.py

```diff
@@ -392,11 +395,12 @@ def size_attachments(self):
     @property
     def replies(self):
         """ returns a list of strings """
-        fs_reply_path = join(self.fs_replies_path, 'message_001.txt')
-        if exists(fs_reply_path):
-            return [json.load(open(fs_reply_path, 'r'))]
-        else:
-            return []
+        reply_list = []
+        for fs_reply in sorted(fnmatch.filter(listdir(self.fs_replies_path), 'message_*.txt')):
```

Comment:
das ganze funktioniert so nicht, wenn ncoh keine replies vorliegen.

```suggestion
        if exists(self.fs_replies_path):
            for fs_reply in sorted(fnmatch.filter(listdir(self.fs_replies_path), 'message_*.txt')):
```